extensions/mssql/src/ui/dialogBase.ts

```diff
@@ -147,6 +147,21 @@ export abstract class DialogBase<DialogResult> {
 		return this.createInputBox(ariaLabel, textChangeHandler, value, enabled, 'password', width);
 	}
 
+	protected createTextInputBox(ariaLabel: string, textChangeHandler: (newValue: string) => Promise<void>, value: string = '', properties: azdata.InputBoxProperties, customValidation?: () => Promise<boolean>): azdata.InputBoxComponent {
```

Comment:
Why is a second InputBox creation method needed here?

Reply:
I didn't know that SQL object names accepted other than alphanumeric values and thought it would be useful to have a generic createTextInputBox that potentially other dialogs could use + I didn't want to overload the generic createInputBox method with more parameters, because @Charles-Gagnon suggested in a previous PR to modify it to only accept an azdata.InputboxProperties instead of passing each parameter individually.

Reply:
I think we should just combine them and remove the default validation. If there are "common" validations that are being done we can just declare them at the module level and pass them in as needed.